projects/plugins/jetpack/extensions/shared/components/stripe-nudge/style.scss

```diff
@@ -15,3 +15,7 @@
 	margin-right: 16px;
 	padding: 6px;
 }
+
+.jetpack-stripe-nudge__banner span.block-editor-warning__action a {
```

Comment:
Is it possible to add custom class to the button, because `span.block-editor-warning__action a` could get changed over time in Gutenberg?

Reply:
Thanks for the suggestion! I couldn't find a way to target the `span`, however, I added a CSS class to the link and made the CSS more robust. Let me know if that works for you